src/main/resources/application.properties

```diff
@@ -37,7 +37,7 @@ lti13.demoMode=false
 lti13.enableAGS=false
 lti13.enableMembership=false
 lti13.enableDynamicRegistration=true
-lti13.enableDeepLinking=false
+lti13.enableDeepLinking=true
```

Comment:
Could you please make this change in application-aws.properties as well?

Reply:
Of course, done.